packaging/generate_rpm.sh

```diff
@@ -0,0 +1,57 @@
+#! /bin/bash -e
+
+PREFIX="usr"
+SPREFIX=${PREFIX}
+SUBPREFIX="opt/${PROJECT}/${VERSION}"
+SSUBPREFIX="opt\/${PROJECT}\/${VERSION}"
+RELEASE="${VERSION_SUFFIX}"
+
+# default release to "1" if there is no suffix
+if [[ -z $RELEASE ]]; then
+  RELEASE="1"
+fi
+
+#add PROJECT_PREFIX to avoid conflict with ocaml dune
+PROJECT="$PROJECT_PREFIX-$PROJECT"
```

Comment:
A very brief search on the Internet seems to indicate rpm packages are commonly lowercase. Maybe we want to follow that convention? If so, this line could be switched to:
```sh
PROJECT=$(echo "$PROJECT_PREFIX-$PROJECT" | tr '[:upper:]' '[:lower:]')
```

Reply:
"Package names SHOULD be in lower case and use dashes in preference to underscores. "
See https://fedoraproject.org/wiki/Packaging:Naming#General_Naming

Reply:
Right, when generating deb package it had to be also lower case.
So now I changed the PROJECT_PREFIX and PROJECT to be lower case.